AdyenActions/Components/3DS2/Action handlers/ThreeDS2CoreActionHandler.swift

```diff
@@ -24,6 +24,9 @@ internal class ThreeDS2CoreActionHandler: Component {
 
     /// :nodoc:
     internal var transaction: AnyADYTransaction?
+    
+    /// `threeDSRequestorAppURL` for protocol version 2.2.0 OOB challenges
+    internal var threeDSRequestorAppURL: URL?
```

Comment:
What is the point to add docs here to internal property, but mark `nodocs` for public property on `AdyenActionComponent` ?

Reply:
for us? :-)